evaluator.builder/src/main/java/org/opencds/cqf/cql/evaluator/builder/library/CqlFileLibraryContentProviderFactory.java

```diff
@@ -96,7 +96,7 @@ public LibraryContentProvider create(String url, List<String> headers) {
         }
 
         if (resourceDirectory.getAbsoluteFile().isDirectory()) {
-            return FileUtils.listFiles(resourceDirectory, new String[] { "cql" }, true);
+            return FileUtils.listFiles(resourceDirectory, new String[] { "cql", "json" }, true);
```

Comment:
This is no longer needed given that the translator options are part of the cli options, correct?

Reply:
Yes, missed this revert. Fixed